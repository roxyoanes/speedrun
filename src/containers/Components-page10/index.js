export { default as Navbar } from "./Navbar";
export { default as Landing } from "./Landing";
export { default as About } from "./About";
export { default as Features } from "./Features";
export { default as Screenshots } from "./Screenshots";
export { default as Clients } from "./Clients";
export { default as Packages } from "./Packages";
export { default as Download } from "./Download";
export { default as Footer } from "./Footer";
