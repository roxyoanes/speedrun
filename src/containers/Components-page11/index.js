export { default as Navbar } from "./Navbar";
export { default as Landing } from "./Landing";
export { default as Services } from "./Services";
export { default as About } from "./About";
export { default as Team } from "./Team";

