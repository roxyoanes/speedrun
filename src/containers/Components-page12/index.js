export { default as Home } from "./Home";
export { default as Navbar } from "./Navbar";
export { default as About } from "./About";
export { default as Skills } from "./Skills";
export { default as Projects } from "./Projects";
export { default as Experience } from "./Experience";

