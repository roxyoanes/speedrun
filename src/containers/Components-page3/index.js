export { default as About } from "./About";
export { default as Education } from "./Education";
export { default as Experience } from "./Experience";
export { default as Interests } from "./Interests";
export { default as Skills } from "./Skills";
export { default as Awards } from "./Awards";
export { default as Navbar } from "./Navbar";
