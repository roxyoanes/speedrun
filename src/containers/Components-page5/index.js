export { default as Navbar } from "./Navbar";
export { default as Landing } from "./Landing";
export { default as About } from "./About";
export { default as Services } from "./Services";
export { default as Gallery } from "./Gallery";
export { default as Pricing } from "./Pricing";
export { default as Team } from "./Team";
export { default as Clients } from "./Clients";
export { default as Contact } from "./Contact";
export { default as Footer } from "./Footer";
