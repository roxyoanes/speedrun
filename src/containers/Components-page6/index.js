export { default as First } from "./First";
export { default as About } from "./About";
export { default as Services } from "./Services";
export { default as Portfolio } from "./Portfolio";
export { default as Pricing } from "./Pricing";
export { default as Contact } from "./Contact";
