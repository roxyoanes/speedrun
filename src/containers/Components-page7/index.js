export { default as Navbar } from "./Navbar";
export { default as Landing } from "./Landing";
export { default as About } from "./About";