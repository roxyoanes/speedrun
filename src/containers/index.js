/* eslint-disable import/prefer-default-export */
export { default as Home } from "./Home";

export { default as FirstPage } from "./FirstPage";
export { default as SecondPage } from "./SecondPage";
export { default as ThirdPage } from "./ThirdPage";
