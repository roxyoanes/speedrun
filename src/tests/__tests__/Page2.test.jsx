import React from "react";
import renderer from "react-test-renderer";
import { Page2 } from "../../containers";

it("matches snapshot", () => {
  const wrapper = renderer.create(
    <Page2 />,
  );
  expect(wrapper).toMatchSnapshot();
});
