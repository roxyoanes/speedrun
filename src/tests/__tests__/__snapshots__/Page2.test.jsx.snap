// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="secondpage"
>
  <div
    className="container"
  >
    <div
      className="navbar"
    >
      <a
        className="start"
        href="#top"
      >
        Start Bootstrap
      </a>
      <div
        className="navbar__right-side"
      >
        <button
          className="button button--menu tablet"
          onClick={[Function]}
          type="button"
        >
          MENU
          <i
            className="fas fa-bars"
          />
        </button>
        <nav
          className="navbar__elements"
        >
          <a
            href="#service"
          >
            <button
              className="navbar__navbar-button"
              type="button"
            >
              SERVICES
            </button>
          </a>
          <a
            href="#portfolio"
          >
            <button
              className="navbar__navbar-button"
              type="button"
            >
              PORTFOLIO
            </button>
          </a>
          <a
            href="#about"
          >
            <button
              className="navbar__navbar-button"
              type="button"
            >
              ABOUT
            </button>
          </a>
          <a
            href="#team"
          >
            <button
              className="navbar__navbar-button"
              type="button"
            >
              TEAM
            </button>
          </a>
          <a
            href="#contact"
          >
            <button
              className="navbar__navbar-button"
              type="button"
            >
              CONTACT
            </button>
          </a>
        </nav>
      </div>
    </div>
    <div
      className="landing-page"
    >
      <h2
        className="landing-page__welcome"
      >
        Welcome To Our Studio!
      </h2>
      <h3
        className="landing-page__subtext"
      >
        IT'S NICE TO MEET YOU
      </h3>
      <a
        href="#service"
      >
        <button
          className="button"
          type="button"
        >
          TELL ME MORE
        </button>
      </a>
    </div>
    <div
      className="services-page"
      id="service"
    >
      <h2
        className="services-page__title"
      >
        SERVICES
      </h2>
      <h1
        className="services-page__sub-title"
      >
        Lorem ipsum dolor sit amet consectetur.
      </h1>
      <div
        className="icons"
      >
        <div
          className="card-icon"
        >
          <span
            className="fa-stack fa-2x"
          >
            <i
              className="fas fa-circle fa-stack-2x"
            />
            <i
              className="fas fa-shopping-cart fa-stack-1x fa-inverse"
            />
          </span>
          <h4
            className="card-icon__icons-title"
          >
            E-Commerce
          </h4>
          <p
            className="card-icon__p-card"
          >
            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima maxime quam architecto quo inventore harum ex magni, dicta impedit.
          </p>
        </div>
        <div
          className="card-icon"
        >
          <span
            className="fa-stack fa-2x"
          >
            <i
              className="fas fa-circle fa-stack-2x"
            />
            <i
              className="fas fa-laptop fa-stack-1x fa-inverse"
            />
          </span>
          <h4
            className="card-icon__icons-title--two"
          >
            Responsive Design
          </h4>
          <p
            className="card-icon__p-card"
          >
            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima maxime quam architecto quo inventore harum ex magni, dicta impedit.
          </p>
        </div>
        <div
          className="card-icon"
        >
          <span
            className="fa-stack fa-2x"
          >
            <i
              className="fas fa-circle fa-stack-2x"
            />
            <i
              className="fas fa-lock fa-stack-1x fa-inverse"
            />
          </span>
          <h4
            className="card-icon__icons-title"
          >
            Web Security
          </h4>
          <p
            className="card-icon__p-card"
          >
            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima maxime quam architecto quo inventore harum ex magni, dicta impedit.
          </p>
        </div>
      </div>
    </div>
    <div
      className="portfolio"
      id="portfolio"
    >
      <h2
        className="services-page__title"
      >
        PORTFOLIO
      </h2>
      <h1
        className="services-page__sub-title"
      >
        Lorem ipsum dolor sit amet consectetur.
      </h1>
      <div
        className="portfolio-images"
      >
        <div
          className="portfolio-images__wrapper"
        >
          <figure
            className="portfolio-images__hover-image"
          >
            <img
              alt=""
              className="portfolio-images__img"
              onClick={[Function]}
              role="presentation"
              src="pic1.jpg"
            />
            <i
              className="fas fa-plus fa-3x"
            />
          </figure>
          <h2
            className="portfolio-images__title-img"
          >
            Threads
          </h2>
          <h1
            className="portfolio-images__sub-title-img"
          >
            Illustration
          </h1>
        </div>
        <div
          className="portfolio-images__wrapper"
        >
          <figure
            className="portfolio-images__hover-image"
          >
            <img
              alt=""
              className="portfolio-images__img"
              onClick={[Function]}
              role="presentation"
              src="pic2.jpg"
            />
            <i
              className="fas fa-plus fa-3x"
            />
          </figure>
          <h2
            className="portfolio-images__title-img"
          >
            Explore
          </h2>
          <h1
            className="portfolio-images__sub-title-img"
          >
            Graphic Design
          </h1>
        </div>
        <div
          className="portfolio-images__wrapper"
        >
          <figure
            className="portfolio-images__hover-image"
          >
            <img
              alt=""
              className="portfolio-images__img"
              onClick={[Function]}
              role="presentation"
              src="pic3.jpg"
            />
            <i
              className="fas fa-plus fa-3x"
            />
          </figure>
          <h2
            className="portfolio-images__title-img"
          >
            Finish
          </h2>
          <h1
            className="portfolio-images__sub-title-img"
          >
            Identity
          </h1>
        </div>
        <div
          className="portfolio-images__wrapper"
        >
          <figure
            className="portfolio-images__hover-image"
          >
            <img
              alt=""
              className="portfolio-images__img"
              onClick={[Function]}
              role="presentation"
              src="04.jpg"
            />
            <i
              className="fas fa-plus fa-3x"
            />
          </figure>
          <h2
            className="portfolio-images__title-img"
          >
            Lines
          </h2>
          <h1
            className="portfolio-images__sub-title-img"
          >
            Branding
          </h1>
        </div>
        <div
          className="portfolio-images__wrapper"
        >
          <figure
            className="portfolio-images__hover-image"
          >
            <img
              alt=""
              className="portfolio-images__img"
              onClick={[Function]}
              role="presentation"
              src="05.jpg"
            />
            <i
              className="fas fa-plus fa-3x"
            />
          </figure>
          <h2
            className="portfolio-images__title-img"
          >
            Southwest
          </h2>
          <h1
            className="portfolio-images__sub-title-img"
          >
            Web Design
          </h1>
        </div>
        <div
          className="portfolio-images__wrapper"
        >
          <figure
            className="portfolio-images__hover-image"
          >
            <img
              alt=""
              className="portfolio-images__img"
              onClick={[Function]}
              role="presentation"
              src="06.jpg"
            />
            <i
              className="fas fa-plus fa-3x"
            />
          </figure>
          <h2
            className="portfolio-images__title-img"
          >
            Window
          </h2>
          <h1
            className="portfolio-images__sub-title-img"
          >
            Photography
          </h1>
        </div>
      </div>
    </div>
    <div
      className="about"
      id="about"
    >
      <h2
        className="about__title"
      >
        ABOUT
      </h2>
      <h1
        className="about__sub-title"
      >
        Lorem ipsum dolor sit amet consectetur.
      </h1>
      <div
        className="timeline"
      >
        <div
          className="container1 card"
        >
          <div
            className="timeline__content-1"
          >
            <div
              className="timeline__circle"
            >
              <img
                alt=""
                className="timeline__img"
                src="mountain.jpg"
              />
            </div>
            <div
              className="timeline__text-box"
            >
              <h4
                className="timeline__h4"
              >
                2009-2011
              </h4>
              <h3
                className="timeline__h3"
              >
                Our Humble Beginnings
              </h3>
              <p
                className="timeline__p"
              >
                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt ut voluptatum eius sapiente, totam reiciendis temporibus qui quibusdam, recusandae sit vero unde, sed, incidunt et ea quo dolore laudantium consectetur!
              </p>
            </div>
          </div>
        </div>
        <div
          className="container2 card"
        >
          <div
            className="timeline__content-2"
          >
            <div
              className="timeline__circle"
            >
              <img
                alt=""
                className="timeline__img"
                src="airplane.jpg"
              />
            </div>
            <div
              className="timeline__text-box"
            >
              <h4
                className="timeline__h4"
              >
                March 2011
              </h4>
              <h3
                className="timeline__h3"
              >
                An Agency is Born
              </h3>
              <p
                className="timeline__p"
              >
                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt ut voluptatum eius sapiente, totam reiciendis temporibus qui quibusdam, recusandae sit vero unde, sed, incidunt et ea quo dolore laudantium consectetur!
              </p>
            </div>
          </div>
        </div>
        <div
          className="container3 card"
        >
          <div
            className="timeline__content-3"
          >
            <div
              className="timeline__circle"
            >
              <img
                alt=""
                className="timeline__img"
                src="laptop.jpg"
              />
            </div>
            <div
              className="timeline__text-box"
            >
              <h4
                className="timeline__h4"
              >
                December 2012
              </h4>
              <h3
                className="timeline__h3"
              >
                Transition to Full Service
              </h3>
              <p
                className="timeline__p"
              >
                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt ut voluptatum eius sapiente, totam reiciendis temporibus qui quibusdam, recusandae sit vero unde, sed, incidunt et ea quo dolore laudantium consectetur!
              </p>
            </div>
          </div>
        </div>
        <div
          className="container4 card"
        >
          <div
            className="timeline__content-4"
          >
            <div
              className="timeline__circle"
            >
              <img
                alt=""
                className="timeline__img"
                src="mountain2.jpg"
              />
            </div>
            <div
              className="timeline__text-box"
            >
              <h4
                className="timeline__h4"
              >
                July 2014
              </h4>
              <h3
                className="timeline__h3"
              >
                Phase Two Expansion
              </h3>
              <p
                className="timeline__p"
              >
                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt ut voluptatum eius sapiente, totam reiciendis temporibus qui quibusdam, recusandae sit vero unde, sed, incidunt et ea quo dolore laudantium consectetur!
              </p>
            </div>
          </div>
        </div>
        <div
          className="container5 card"
        >
          <div
            className="timeline__circle timeline__circle--last"
          >
            <h4
              className="timeline__last"
            >
              Be Part
               
              <br />
              Of Our
               
              <br />
              Story!
            </h4>
          </div>
        </div>
      </div>
    </div>
    <div
      className="team-page"
      id="team"
    >
      <h2
        className="team-page__title"
      >
        OUR AMAZING TEAM
      </h2>
      <h1
        className="team-page__sub-title"
      >
        Lorem ipsum dolor sit amet consectetur.
      </h1>
      <div
        className="team-wrapper"
      >
        <div
          className="team-card"
        >
          <img
            alt=""
            className="team-card__img"
            src="person1.jpg"
          />
          <h4
            className="team-card__title"
          >
            Kay Garland
          </h4>
          <p
            className="team-card__p"
          >
            Lead Designer
          </p>
          <div
            className="team-card-icons"
          >
            <span
              className="fa-stack fa-2x"
            >
              <i
                className="fas fa-circle fa-stack-2x"
              />
              <i
                className="fab fa-twitter fa-stack-1x fa-inverse"
              />
            </span>
            <span
              className="fa-stack fa-2x"
            >
              <i
                className="fas fa-circle fa-stack-2x"
              />
              <i
                className="fab fa-facebook-f fa-stack-1x fa-inverse"
              />
            </span>
            <span
              className="fa-stack fa-2x"
            >
              <i
                className="fas fa-circle fa-stack-2x"
              />
              <i
                className="fab fa-linkedin-in fa-stack-1x fa-inverse"
              />
            </span>
          </div>
        </div>
        <div
          className="team-card"
        >
          <img
            alt=""
            className="team-card__img"
            src="person2.jpg"
          />
          <h4
            className="team-card__title"
          >
            Larry Parker
          </h4>
          <p
            className="team-card__p"
          >
            Lead Marketer
          </p>
          <div
            className="team-card-icons"
          >
            <span
              className="fa-stack fa-2x"
            >
              <i
                className="fas fa-circle fa-stack-2x"
              />
              <i
                className="fab fa-twitter fa-stack-1x fa-inverse"
              />
            </span>
            <span
              className="fa-stack fa-2x"
            >
              <i
                className="fas fa-circle fa-stack-2x"
              />
              <i
                className="fab fa-facebook-f fa-stack-1x fa-inverse"
              />
            </span>
            <span
              className="fa-stack fa-2x"
            >
              <i
                className="fas fa-circle fa-stack-2x"
              />
              <i
                className="fab fa-linkedin-in fa-stack-1x fa-inverse"
              />
            </span>
          </div>
        </div>
        <div
          className="team-card"
        >
          <img
            alt=""
            className="team-card__img"
            src="person3.jpg"
          />
          <h4
            className="team-card__title"
          >
            Diana Pertersen
          </h4>
          <p
            className="team-card__p"
          >
            Lead Developer
          </p>
          <div
            className="team-card-icons"
          >
            <span
              className="fa-stack fa-2x"
            >
              <i
                className="fas fa-circle fa-stack-2x"
              />
              <i
                className="fab fa-twitter fa-stack-1x fa-inverse"
              />
            </span>
            <span
              className="fa-stack fa-2x"
            >
              <i
                className="fas fa-circle fa-stack-2x"
              />
              <i
                className="fab fa-facebook-f fa-stack-1x fa-inverse"
              />
            </span>
            <span
              className="fa-stack fa-2x"
            >
              <i
                className="fas fa-circle fa-stack-2x"
              />
              <i
                className="fab fa-linkedin-in fa-stack-1x fa-inverse"
              />
            </span>
          </div>
        </div>
      </div>
      <p
        className="team-page__p"
      >
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut eaque, laboriosam veritatis, quos non quis ad perspiciatis, totam corporis ea, alias ut unde.
      </p>
    </div>
    <div
      className="sponsors"
    >
      <img
        alt=""
        className="img"
        src="envato.jpg"
      />
      <img
        alt=""
        className="img"
        src="designmodo.jpg"
      />
      <img
        alt=""
        className="img"
        src="themeforest.jpg"
      />
      <img
        alt=""
        className="img"
        src="creative-market.jpg"
      />
    </div>
    <div
      className="contact"
      id="contact"
    >
      <h2
        className="contact__title"
      >
        CONTACT US
      </h2>
      <h1
        className="contact__sub-title"
      >
        Lorem ipsum dolor sit amet consectetur.
      </h1>
      <div
        className="inputs"
      >
        <div
          className="inputs__left"
        >
          <input
            className="inputs__name input"
            placeholder="Your Name*"
            type="text"
          />
          <input
            className="inputs__email input"
            placeholder="Your Email*"
            type="text"
          />
          <input
            className="inputs__phone input"
            placeholder="Your Phone*"
            type="text"
          />
        </div>
        <textarea
          className="inputs__message input"
          placeholder="Your Message*"
          type="text"
        />
      </div>
      <div
        className="button-btn"
      >
        <button
          className="button"
          type="button"
        >
          SEND MESSAGE
        </button>
      </div>
    </div>
    <div
      className="footer"
    >
      <p
        className="footer__rights"
      >
        Copyright © Your Website 2019
      </p>
      <div
        className="footer__icons"
      >
        <span
          className="fa-stack fa-2x"
        >
          <i
            className="fas fa-circle fa-stack-2x"
          />
          <i
            className="fab fa-twitter fa-stack-1x fa-inverse"
          />
        </span>
        <span
          className="fa-stack fa-2x"
        >
          <i
            className="fas fa-circle fa-stack-2x"
          />
          <i
            className="fab fa-facebook-f fa-stack-1x fa-inverse"
          />
        </span>
        <span
          className="fa-stack fa-2x"
        >
          <i
            className="fas fa-circle fa-stack-2x"
          />
          <i
            className="fab fa-linkedin-in fa-stack-1x fa-inverse"
          />
        </span>
      </div>
      <div
        className="footer__right-side"
      >
        <a
          className="footer__right-side__a"
          href="#footer"
        >
          Privacy Policy
           
        </a>
        <a
          className="footer__right-side__a"
          href="#footer"
        >
          Terms of Use
        </a>
      </div>
    </div>
  </div>
</div>
`;
